atm.js

```diff
@@ -1,21 +1,57 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-}
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
-}
-
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+//checking
+var checkingDepoButt = $("#checking_deposit")
+var checkingWithdrawButt = $("#checking_withdraw")
+var checkingInput = $("#checking_amount")
+var checkingBalance = $("#checking_balance")
+var checkingTotal = 0
+//savings
+var savingsDepoButton = $("#savings_deposit")
+var savingsWithdrawButt = $("#savings_withdraw")
+var savingsInput = $("#savings_amount")
+var savingsBalance = $("#savings_balance")
+var savingsTotal = 0
+
```

Comment:
Implementing an OOJS approach could help you condense a lot of your code. 

Think about what an Account constructor function would look like. Passing in the DOM representation of an account (i.e., a `<div>` with the class `account`) as an argument to the constructor would allow you to target the buttons, inputs, etc. that belong to a given account without having to hardcore separate variables for your checking and savings accounts.